app/assets/stylesheets/main.css

```diff
@@ -1,11 +1,19 @@
 .main {
-  width: 83.5%;
+  width: 100%;
```

Comment:
- [ ] According to the project requirements for 2 people:

![image](https://github.com/JuanDBta/booking-bites/assets/57421823/ffa7459e-c3c1-4df3-811d-d1fea6b1d155)

The website is not responsive for the mobile view and it's impossible to navigate:


![image](https://github.com/JuanDBta/booking-bites/assets/57421823/7f84bd58-81e4-4ff1-ba16-c2c03401fda1)

![image](https://github.com/JuanDBta/booking-bites/assets/57421823/c172b2dd-d6a2-40d9-8735-7759fd906c3e)

Therfore, kindly make it so the page is responsive, and the mobile view is fully implemented.

That way, we will meet this particular [requirement](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md#project-requirements). 👍